.travis.yml

```diff
@@ -21,9 +21,9 @@ script:
 
 deploy:
   provider: npm
-  email: simonc@linux.com
+  email: frank@cozycloud.cc
   api_key:
-    secure: AfkB9NoLAyhdsA/0KYQxMRapABvawz8a5UATs6yWO7Nh/ngwJQ5eBOXZo0oo4RqpdploYG6AHI5PyNb5FDV4IvI92oFy+MrdPefw7AgQXP+qwiGuwoxs5KnUM6DD5s+F0ClJbtMoWaRCdkD3HXNmCPeM9eDrIbD3scqXiH9wmK2KBD4M+7TKpJ68mL8zd+oE3Lfv4SQiE7PjTMP4Iy6JsNcJFHNp2/QTW9drGOlPSnJTY4Usllc7caaJ6iBJ2EoWNxZ+JhxpqHpPHt2CXNeSltLltw84C2179aiXhKSx7EZwWb85QK3JgGwRWBorRnUPIgltVxIc/Otr2y4mgWw/A1XWqqifEC0PtZoqBJEc13A8YmyIXxKnO2QcTl5sJVifjc9/EEXz9e4Xsmd5u7xlRk/KC7Kh2fOhgatekarjsdBxHSJqtAstd+6xEBQHwS5IKFGDl5iGyTzc7yKoQH/SdoeWIp688YmKrgqhMWc5+c58uvSQn2AjcUyDtG433CMK01yImEY24luk56kL8YGequoznWKsm5NU3PZ9UOENtHrbkcaOmgLp216VxeaQ5BN7/fnJ57Uh0beiPqSwMF88R5RWM8jDlzOgMwS5vRrbOqZp8I0LfiLNBrxJr9xd7STT/Rk6bwySNBw7y2c4TYGp0gZmhjLHN1KH4UoARLPiK4w=
+    secure: Ci6JiHjsEQPUHLKVuWvfMLvk/Obs3vqMMig91loV/JPgrVInX6s4YSmeKnxMjidAIWjwnqEVJrNUn9v2kvxTUI2/ifAapnq9XtxNV8rhn8ZLku3bUvR/8rRvsIgDXvl4ziNT7l0zN44WzoZNwhNtON3F5OIwiBAc3BQjnYBIbao8wNb4v3KXyuS8r/ABcVx5smAnoGbpCj8zTkCzfj0liPwWNwQy1anu6r8BWNFCaviCwLT1JbfsxFliFMBdn5N9fETToN+EMmai7oZEFgf8lPXILl5aQl75+Xm7LloqD+U2S+cpbQyze34WFitW/t9FD5ZRD4vlLps7rlAIju3ErbORzHZRLP3saf1fH7lp0Ax1jMjpcJV20PoF3z7NoyDaeg3bkKagG8Qmf1wbCKrBKjKxPcolVizA+vmyc15AjQTWp4Dcs48OhQIc9Fcs7YjZ97QYYTziAvG93fgqh7XSs4JFLLtXTRG1WkCKaYBOgHAaTOBFHiRSr9D+zdU6VwE++VJuI2T2qXTAcTyWZwygJotKyfHCXIjNirtJ9Ke8t325rSqSCSTGnN59nHXDsLA2qEq2O4unfosKY6TYvBD/RQaLtO2tyeDWu2TTCJ95MqJPjYvOBgQDRG7meOp1gX2SDwx/GAtekun6F6kxBLdf8tyqKnQhu1UIgBvziTlN9a8=
```

Comment:
Not worry, it's an encrypted key with the travis cli. You can see the documentation [here](https://docs.travis-ci.com/user/deployment/npm/).